src/libsystemd-network/network-internal.c

```diff
@@ -272,25 +273,35 @@ int config_parse_hwaddr(const char *unit,
                         void *userdata) {
         struct ether_addr **hwaddr = data;
         struct ether_addr *n;
-        int r;
+        int r, before = 0, after = 0;
+        const char *start = rvalue, *end;
 
         assert(filename);
         assert(lvalue);
         assert(rvalue);
         assert(data);
 
+        while (isspace(*start))
+                start++;
```

Comment:
we usually write this as: 
```c
start += strspn(start, WHITESPACE);
```
Not that it really matters, but it is a bit shorter, and also, well, faster (SSE optimization, yay!), which of course is pretty irrelevant here...